.golangci.yml

```diff
@@ -14,3 +18,4 @@ linters:
     - deadcode
     - staticcheck
     - gosec
+    - goimports
```

Comment:
Do we still need to enable gofmt as goimports also formats the file?
Following is from https://godoc.org/golang.org/x/tools/imports:

>func Process
func Process(filename string, src []byte, opt *Options) ([]byte, error)
Process formats and adjusts imports for the provided file. If opt is nil the defaults are used, and if src is nil the source is read from the filesystem.

Reply:
I do agree that goimports does everything gofmt does. That being said, I don't think there is much harm in keeping it in the list. Makes it explicit that we are running this linters, even if it means we run it twice. Or we can remove it but add a comment to the yml file to indicate that goimports supersedes gofmt.

Reply:
@srikartati thanks for pointing it out, I didn't know goimports supersedes gofmt. I also think there is no much harm to keep gofmt explicitly as it's more well known and widely adopted?

Reply:
Sure.. format can be done two times. I was looking at the API and found out about it--thought I will check with you.